commands/foundry.js

```diff
@@ -0,0 +1,184 @@
+const { findNetwork } = require('../helpers');
+const which = require('which');
+
+const log = require('debug')('synpress:foundry');
+
+let activeChains;
+
+module.exports = {
+  async resetState() {
+    log('Resetting state of foundry');
+    activeChains = undefined;
+  },
+  async getActiveChains() {
+    return activeChains;
+  },
+  async forkChains(options) {
+    await validateIfAnvilIsInstalledOrThrow();
+
+    if (typeof options === 'object') {
+      const chains = await module.exports.runAnvilWithViem(
+        options.chainsToFork,
+      );
+
+      return { chains };
+    } else if (typeof options === 'string') {
+      if (isNaN(options)) {
+        // todo: add support for:
+        // (multiple) network IDs
+        // (single) network name
+        // (multiple) network names
+      } else {
+        // todo: add support for:
+        // (single) network ID
+      }
+    }
+  },
+  async setupViem(anvilChainType) {
+    try {
+      const {
+        createTestClient,
+        createPublicClient,
+        createWalletClient,
+        http,
+      } = require('viem');
```

Comment:
```suggestion
      } = await import('viem');
```
This should work without even converting it to cjs